.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: ruby
+sudo: required
+cache: bundler
+rvm:
+- 2.4.5
+services:
+- docker
+addons:
+  postgresql: '9.5'
+env:
+  global:
+  - DATABASE_USERNAME=root
+  - DATABASE_PASSWORD=smartvm
+before_install:
+- gem install bundler
+- psql -c "CREATE USER $DATABASE_USERNAME SUPERUSER PASSWORD '$DATABASE_PASSWORD';" -U postgres
```

Comment:
Looking at the other repos, I like this  option best for services that commit a `database.yml`, Especially since we're DRYer with the substituted environment variables.  If we're not committing a `database.yml` the `DATABASE_URL` makes sense because that is the single declaration of `username`, `password` as well as all the other connection params.

Reply:
> If we're not committing a database.yml

I don't think you should ever be committing a `database.yml` ...

Reply:
So, that's the general idea behind what service_portal did.  If that's *not* the direction we want to go I'm fine with that; we just need to be consistent.